docs/configure.md

```diff
@@ -193,6 +197,7 @@ Available elements are:
 | ResultsDirectory  | string | Directory for test run reports. E.g. trx, coverage etc.                                         |
 | SolutionDirectory | string | Working directory for test invocation. Results directory can be relative to this. Used by IDEs. |
 | MaxCpuCount       | int    | Degree of parallelization, spawns `n` test hosts to run tests. Default: 1. Max: Number of cpu cores. |
+| TestSessionTimeout | int   | Testplatform will canceal the test run after it exceeded given TestSessionTimeout and will show the results of tests which ran till that point. |
```

Comment:
Please add a note mentioning the version where this supported. E.g. `Version: 15.5+`.